TFT/src/User/API/FanControl.c

```diff
@@ -95,3 +95,22 @@ void ctrlFanQuery(void)
     ctrlFanQueryWait = storeCmd("M710\n");
   }
 }
+
+#define RRF_FAN_QUERY_MS  3000
```

Comment:
if I can suggest, I would provide the #define at the beginning of the file close to other #define